gensim/test/test_corpora.py

```diff
@@ -400,6 +402,73 @@ def test_indexing(self):
         pass
 
 
+class TestWikiCorpus(TestTextCorpus):
```

Comment:
One of your tests "stuck" in Travis, please check, what's a problem and fix https://travis-ci.org/RaRe-Technologies/gensim/jobs/322641221#L828

Reply:
Also, question: why you need `TextTextCorpus` here?

Reply:
I am inheriting so that we make sure the `WikiCorpus` class passes not only its own tests but also the one's defined for the `TextCorpus` class (since` WikiCorpus` inherits from `TextCorpus`). Else we can inherit from `CorpusTestCase` or even from `unittest.Testcase`